grid/recruitment/create-candidate/profile.supr

```diff
@@ -3,7 +3,7 @@ Create Candidate
 Create new candidate in ATS (Applicant Tracking System).
 """
 name = "recruitment/create-candidate"
-version = "2.0.0"
+version = "2.1.0"
```

Comment:
todo: add breezy-hr and workable maps for recruitment/create-candidate@2.1 profile

Reply:
What do you mean? The changes are backwards compatible, and the maps have already had their versions bumped

Reply:
My mistake, I did not notice that maps versions are already bumped.